google/gce/gce-how-to/deploy-kubernetes-with-ubuntu-pro.rst

```diff
@@ -0,0 +1,244 @@
+Deploy Kubernetes with Ubuntu Pro on GCE
+========================================
+
+
+Limitations
+-----------
+
+Google does not yet have Ubuntu Pro image offerings for the GKE nodes i.e. you cannot choose
```

Comment:
People might not know what GKE is

> image offerings for the GKE nodes

image offerings for the GKE (Google Kubernetes Engine) nodes

Reply:
Added a link as well! Thank you.